Tests/Functional/Frontend/TeaIndexTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Tea\Tests\Functional\Frontend;
+
+use TTN\Tea\Frontend\TeaIndex;
+use TYPO3\TestingFramework\Core\Functional\Framework\Frontend\InternalRequest;
+
+/**
+ * @coversNothing
+ */
+final class TeaIndexTest extends AbstractTest
+{
+    /**
+     * @test
+     */
+    public function rendersAllAvailableTeas(): void
+    {
+        $this->importCSVDataSet(__DIR__ . '/Fixtures/Database/ContentElementTeaIndex.csv');
+        $this->importCSVDataSet(__DIR__ . '/Fixtures/Database/Teas.csv');
+
+        $request = new InternalRequest();
+        $request = $request->withPageId(1);
+
+        $html = $this->executeFrontendRequest($request)->getBody()->__toString();
+
+        self::assertStringContainsString('Godesberger Burgtee', $html);
+        self::assertStringContainsString('Earl Grey', $html);
+        self::assertStringContainsString('Oolong', $html);
```

Comment:
For this testcase to be minimal, two teas should suffice (in the test data and the assertions).